packages/playwright/test/RunPartial/FrameTests.cs

```diff
@@ -25,6 +25,26 @@ public async Task ShouldAnalyzeNestedIFrames()
             Assert.Fail("Could not find label violation");
         }
 
+        [Test]
+        public async Task ShouldAnalyzeInNestedIFrames()
```

Comment:
These new test cases are good additions, but I think still not quite complete enough; none of the new tests are exercising the library's special case behavior of "if the locator is inside a child frame, the axe-core initiator frame is the locator's frame and not the page's main frame". In addition to the tests you've already added, it would be nice to have test coverage for:

1. Locator refers to an element inside an iframe (eg, repeat `ShouldAnalyzeInNestedIframes` but with locator `Page!.FrameLocator("#ifr-foo").Locator("#foo-bar")` and expecting to find only 1 label violation)"
2. As 1, but for framesets (eg, repeat `ShouldAnalyzeInNestedFramesets` but with locator `Page!.FrameLocator("#frm-foo").Locator("#foo-bar")` and expecting to find only 1 label violation)
3. Locator refers to an element inside a shadow DOM (eg, repeat `ShouldAnalyzeInShadowDOMFrames` but with a locator for selector `#shadow-frame` instead of `#shadow-root` and expecting to find only 1 label violation).